### DISCUSSION TITLE
Does Gitlens have the ability to mimic the -M or -C flags in the manuscript?

### DISCUSSION THREAD

**Discussion by Jaraujo6** [gitkraken/vscode-gitlens#1747] [Q&A]
If I understand the [docs](https://git-scm.com/docs/git-blame#_options) correctly, `-M` and `-C` detect moved or copied lines within a file and assigns the blame to the parent commit when no code has actually changed. '

Gitlens has a whitespace equivalent setting for `-w`: `gitlens.blame.ignoreWhitespace": true`. Do the above flags also have an equivalent setting?

**Comment by charliepalmer-xlnx** [comment]
You can add arbitrary arguments with `gitlens.advanced.blame.customArguments`:
```
// settings.json
{
    "gitlens.advanced.blame.customArguments": [
        "-C", // if the lines come from another file, git will try and find them
    ],
}
```
Then close/open a file to get the blame lines to update. It would be nicer to be able to toggle this setting on/off interactively, perhaps there's a neater solution for this
